File: 2275. Largest Combination With Bitwise AND Greater Than Zero/solution1.java
class Solution {
    public int largestCombination(int[] candidates) {
        int[] freq = new int[25];
        int maxFreq = 0;
        for (int i=0;i<25;i++)
        {
            for (int c:candidates)
            {
                if ((c & (1<<i)) > 0)
                {
                    freq[i]++;
                    maxFreq = Math.max(maxFreq, freq[i]);
                }
            }
        }

        return maxFreq;
    }
}

File: 2275. Largest Combination With Bitwise AND Greater Than Zero/solution1.cpp
class Solution {
public:
    int largestCombination(vector<int>& candidates) {
        vector<int> v(25, 0);
        for (int i=0;i<25;i++)
        {
            for (auto &c:candidates)
            {
                if (c & (1<<i))
                    v[i]++;
            }
        }

        return *max_element(v.begin(), v.end());
    }
};
